docs/release-info/release-notes.md

```diff
@@ -24,6 +24,26 @@ title: "Release notes"
 
 <!--Replace {{DRUIDVERSION}} with the correct Druid version.-->
 
+## Druid 29.0.1
+
+Apache Druid 29.0.1 is a patch release that fixes some issues in the Druid 29.0.0 release.
+
+### Bug fixes
+
+* Added type verification for INSERT and REPLACE to validate that strings and string arrays aren't mixed [#15920](https://github.com/apache/druid/pull/15920)
+* [Concurrent replace](https://druid.apache.org/docs/latest/ingestion/concurrent-append-replace) now allows pending Peon segments to be upgraded using the Supervisor [#15995](https://github.com/apache/druid/pull/15995)
+* Changed the `targetDataSource` attribute to return a string containing the name of the datasource [#16004](https://github.com/apache/druid/pull/16004)
+* Decreased the size of the distribution Docker image [#15968](https://github.com/apache/druid/pull/15968)
+* Fixed an issue with SQL-based ingestion where string inputs, such as from CSV, TSV, or string-value fields in JSON, are ingested as null values when they are typed as LONG or BIGINT [#15999](https://github.com/apache/druid/pull/15999)
+* Fixed an issue where a web console-generated Kafka supervisor spec has `flattenSpec` in the wrong location [#15946](https://github.com/apache/druid/pull/15946)
+* Fixed an issue with filters on expression virtual column indexes incorrectly considering values null in some cases for expressions which translate null values into not null values [#15959](https://github.com/apache/druid/pull/15959)
+* Fixed an issue where the data loader crashes if the incoming data can't be parsed [#15983](https://github.com/apache/druid/pull/15983)
+* Improved DOUBLE type detection in the web console [#15998](https://github.com/apache/druid/pull/15998)
+* Web console-generated queries now only set the context parameter `arrayIngestMode` to `array` when you explicitly opt in to use arrays [#15927](https://github.com/apache/druid/pull/15927)
+* The web console now displays the results of an MSQ query that writes to an external destination through the [`EXTERN`](https://druid.apache.org/docs/latest/multi-stage-query/reference#extern-function) function [#15969](https://github.com/apache/druid/pull/15969)
+
```

Comment:
We should also capture this pr : https://github.com/apache/druid/pull/15931
Postgresql dependency bumped up from xxx to xxx